crates/re_log_types/src/component_types/tensor.rs

```diff
@@ -535,7 +551,7 @@ impl<'a> TryFrom<&'a Tensor> for ::ndarray::ArrayViewD<'a, half::f16> {
 
 #[cfg(feature = "image")]
 #[derive(thiserror::Error, Debug)]
-pub enum ImageError {
+pub enum TensorImageError {
```

Comment:
Differentiating this from the image::ImageError had me scratching my head for a bit.